perl/bin/battenberg.pl

```diff
@@ -441,6 +445,7 @@ =head1 SYNOPSIS
     -platform              -pl  Sequencing platfrom [ILLUMINA]
     -genderloci            -gl  List of gender loci, required when '-ge L' [share/gender/GRCh37d5_Y.loci]
                                 - these are loci that will not present at all in a female sample
+    -seed                  -se Integer value to be used as a seed for random number generation
```

Comment:
Can you indicate the default at the end of the text as perl general convention `[1488823153]`